### PR TITLE
add profile photo component

### PR DESCRIPTION
**Pull request by ibsa-ejeta**
### What this PR does (required):
- adds a new component which displays the profile photo
### Screenshots / Videos (required):
ticket22

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: client/src/components/ProfilePhoto/ProfilePhoto.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            className={classes.photoButton}
+            classes={{
+              root: classes.buttonRoot,
+            }}
+          >
+            Upload a file from your device
+          </Button>
+        </CardActions>
+        <Typography variant="subtitle1" className={classes.photoText}>
+          <i className="fas fa-trash-alt fa-fw"></i> Delete photo
```
Comment (rajivtitus) [inline client/src/components/ProfilePhoto/ProfilePhoto.tsx:41]: Please resort to using Material UI icons ( the dependency should already be installed) as opposed to font awesome.

Ref: https://material-ui.com/components/material-icons/

File: client/src/components/ProfilePhoto/useStyles.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    alignItems: 'center',
+    paddingBottom: '2rem',
+    textAlign: 'center',
+    width: '50%',
+    height: '70vh',
+  },
+
+  photoCard: {
+    width: '35%',
+    marginLeft: 'auto',
```
Comment (rajivtitus) [inline client/src/components/ProfilePhoto/useStyles.ts:17]: Maybe consider using CSS short hand notation here :)

File: client/src/components/ProfilePhoto/useStyles.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    marginRight: 'auto',
+    marginTop: '2rem',
+    border: 'none',
+    boxShadow: 'none',
+    alignItems: 'center',
+    justifyContent: 'center',
+    height: '60vh',
+  },
+  photoAvatar: {
+    marginLeft: 'auto',
```
Comment (rajivtitus) [inline client/src/components/ProfilePhoto/useStyles.ts:27]: Same as comment above

File: client/src/components/ProfilePhoto/ProfilePhoto.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { User } from '../../interface/User';
+import useStyles from './useStyles';
+import { Button, Card, Typography, Avatar, CardActions, Paper } from '@material-ui/core';
+
+interface Props {
+  loggedInUser: User;
+}
+
+const ProfilePhoto = ({ loggedInUser }: Props): JSX.Element => {
```
Comment (rajivtitus) [inline client/src/components/ProfilePhoto/ProfilePhoto.tsx:9]: You can access the loggedInUser from context as opposed to passing down as props

**Review by rajivtitus — APPROVED** [review]
Good work! Just some minor comments, please address before merging. Thank you!

File: client/src/components/ProfilePhoto/ProfilePhoto.tsx (hunk 1 above)
Comment (ibsa-ejeta) [reply client/src/components/ProfilePhoto/ProfilePhoto.tsx]: replace font awesome with material ui icon

File: client/src/components/ProfilePhoto/useStyles.ts (hunk 2 above)
Comment (ibsa-ejeta) [reply client/src/components/ProfilePhoto/useStyles.ts]: used shorthand

File: client/src/components/ProfilePhoto/useStyles.ts (hunk 3 above)
Comment (ibsa-ejeta) [reply client/src/components/ProfilePhoto/useStyles.ts]: used shorthand

File: client/src/components/ProfilePhoto/ProfilePhoto.tsx (hunk 4 above)
Comment (ibsa-ejeta) [reply client/src/components/ProfilePhoto/ProfilePhoto.tsx]: accessed loggedInUser directly in the component

**Pull request merged**